extendedspans/src/main/kotlin/me/saket/extendedspans/ExtendedSpans.kt

```diff
@@ -47,6 +48,16 @@ class ExtendedSpans(
       text.getTtsAnnotations(start = 0, end = text.length).fastForEach {
         addTtsAnnotation(it.item, it.start, it.end)
       }
+      @Suppress("DEPRECATION")
+      text.getUrlAnnotations(start = 0, end = text.length).fastForEach {
+        addUrlAnnotation(it.item, it.start, it.end)
+      }
+      text.getLinkAnnotations(start = 0, end = text.length).fastForEach { range ->
```

Comment:
TIL about link annotations. Are they the reason why a beta version of compose is needed?

Reply:
Ya, they were added in the 1.7.0 beta. https://developer.android.com/reference/kotlin/androidx/compose/ui/text/LinkAnnotation

Reply:
got it, thank you!

Reply:
I'll probably have to split the changes and make two releases, one for 1.6.x and one for 1.7.x-beta